MAUI/DataForm/getting-started.md

```diff
@@ -7,53 +7,40 @@ control: SfDataForm
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI DataForm (SfDataForm)
+# Getting Started with the .NET MAUI DataForm
 
 This section provides a quick overview of how to get started with the [.NET MAUI DataForm(SfDataForm)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.html) for .NET MAUI and a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
 
-## Creating an application using the .NET MAUI DataForm
+Follow the steps below to add a basic dataForm view to your project.
```

Comment:
add in above paragraph - refer this pr
https://github.com/syncfusion-content/maui-docs/pull/2656/files

Reply:
Updated the changes